qutip/metrics.py

```diff
@@ -323,9 +319,7 @@ def hellinger_dist(A, B, sparse=False, tol=0):
     --------
     >>> x=fock_dm(5,3)
     >>> y=coherent_dm(5,1)
-    >>> hellinger_dist(x,y)
-    1.3725145002591095
-
+    >>> np.testing.assert_almost_equal(hellinger_dist(x,y), 1.3725145002591095)
```

Comment:
I did not try it out individually, only changed it to make it more robust. I am also thinking how to avoid using numpy testing for these kinds of tests cause as numpy and qutip is updated a lot of doctest style tests might keep failing because they don't work in an approximate fashion.